### DISCUSSION TITLE
Naming of Feature Xaml Pages

### DISCUSSION THREAD

**Discussion by acantley** [adesokan01/T24S2T1EXTG01#2] [General]
I have begun development on the Mortgage Calculator under **feature_mortgage_calc** and have named the XAML file for the page MortgageCalc.xaml. Are we satisfied with this name? By that naming convention, the Currency Converter Calculator page could be called CurrencyCalc.xaml. Please advise your thoughts.

**Comment by adesokan01** [comment]
Hi Adam,

I have not been on my computer since morning. The reason while I am just replying you . The name you gave is fine by me. I will use the nomenclature you suggested for the Currency converter calculator page as well.

Thanks,
Akeem
________________________________
From: Adam Cantley ***@***.***>
Sent: Thursday, September 12, 2024 12:36 PM
To: adesokan01/T24S2T1EXTG01 ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [adesokan01/T24S2T1EXTG01] Naming of Feature Xaml Pages (Discussion #2)


I have begun development on the Mortgage Calculator under feature_mortgage_calc and have named the XAML file for the page MortgageCalc.xaml. Are we satisfied with this name? By that naming convention, the Currency Converter Calculator page could be called CurrencyCalc.xaml. Please advise your thoughts.
